docs/marionette.application.md

```diff
@@ -2,21 +2,34 @@
 
 # Marionette.Application
 
-The `Backbone.Marionette.Application` object is the hub of your composite
-application. It organizes, initializes and coordinates the various pieces of your
-app. It also provides a starting point for you to call into from your HTML
-script block, or directly from your JavaScript files if you prefer to go that
-route.
+The **Application** is a container for the rest of your code. It is recommended
+that every Marionette app have at least one instance of Application.
```

Comment:
If you look through the source, it's easy to confirm that you can have any number of Applications or Routers. The only thing to be wary of is a channel conflict, but changing the channel name is documented, and using the channel is deprecated.